src/core/dpl-cms-api/dpl-cms-api.example.json

```diff
@@ -0,0 +1,83 @@
+{
+  "swagger": "2.0",
+  "schemes": ["http"],
+  "info": {
+    "description": "The REST API provide by the core REST module.",
+    "title": "DPL CMS - REST API",
+    "version": "Versioning not supported"
```

Comment:
Consider if this should be included in VCS.

We use the external url from the DPL CMS repository in our Orval configuration.

Reply:
👍